.github/workflows/python-package.yml

```
@@ -24,14 +23,27 @@ jobs:
         run: |
           pip install setuptools wheel twine build
```

This runner doesn't use twine.
Preexisting, I think you need to use the `-U` param to update setuptools and wheel:
```suggestion
          pip install -U setuptools wheel
          pip install build
```